cmd/query/app/apiv3/http_gateway_test.go

```diff
@@ -90,25 +91,111 @@ func TestHTTPGatewayTryHandleError(t *testing.T) {
 	assert.Contains(t, string(w.Body.String()), e, "writes error message to body")
 }
 
-func TestHTTPGatewayGetTraceErrors(t *testing.T) {
-	gw := setupHTTPGatewayNoServer(t, "")
+func TestHTTPGatewayGetTrace(t *testing.T) {
+	traceId, _ := model.TraceIDFromString("123")
+	testCases := []struct {
+		name          string
+		expectedQuery spanstore.GetTraceParameters
+		params        map[string]string
```

Comment:
if params are the input please move them above the expected values, in both struct declaration and in the initialization below.

Reply:
done